client/Components/GuestDropdown.jsx

```diff
@@ -37,13 +49,17 @@ class GuestDropdown extends React.Component {
   }
 
   render() {
-    const { expand } = this.state;
+    const { adults, children, expand, infants } = this.state;
+    const { maxGuests } = this.props;
     return (
       <div>
         <div className={styles.guests_box}>
```

Comment:
should we make sure this is a form? (since there seem to be form elements nested within it)